app.js

```diff
@@ -18,7 +18,11 @@ app.use(async function(ctx, next) {
             ctx.throw(e.response.status, e.response.text);
         }
 
-        if (e instanceof httpErrors.Forbidden) {
+        if (
+            e instanceof httpErrors.Forbidden ||
```

Comment:
let's not use `instanceof` to check for errors, it's incredibly fragile (see e.g. https://github.com/isaacs/rimraf/commit/aa50e0298f66b3375045e8d9d86992d696a56b09)

Reply:
I was able to use a switch statement based off of `e.status` here, and I like it much more. Thanks!